Exiled.API/Features/Roles/Role.cs

```diff
@@ -146,8 +146,11 @@ public bool Is<T>(out T role)
         /// <inheritdoc/>
         public override bool Equals(object obj) => base.Equals(obj);
 
-        /// <inheritdoc/>
-        public override string ToString() => Type.ToString();
+        /// <summary>
+        /// Returns the role in a human-readable format.
+        /// </summary>
+        /// <returns>A string containing role-related data.</returns>
+        public override string ToString() => $"{Side} {Team} {Type} {IsValid}";
```

Comment:
With this new change you need to change what I told you before in Player::ToString() (```Role.Type``)

Reply:
I prefer it give all this information in Player::ToString

Reply:
Then delete ``{Role?.Team}``

Reply:
Oh yes i have not see it